webapp/crashstats/supersearch/models.py

```diff
@@ -86,8 +86,6 @@ def validate_products(product_value):
 
 
 class SuperSearch(ESSocorroMiddleware):
-    implementation = supersearch.SuperSearch
```

Comment:
Previously, you could define the `implementation` class attribute and the SocorroMiddleware would instantiate the specified class passing in the configman configuration and all that. We no longer have configman configuration, so now each middleware class needs to supply its own `get_implementation` method which configures the socorro class it's using (e.g. ESCrashStorage, SuperSearch, etc).

I changed the code that creates the implementation so it only looked at the `get_implementation` method. Then I went through all the middleware classes and removed `implementation` class attributes replacing them with the appropriate `get_implementation` code.